WordPress/Classes/ViewRelated/Blog/SiteManagement/SettingsTitleSubtitleController.swift

```diff
@@ -258,6 +262,37 @@ extension SettingsTitleSubtitleController {
         }
     }
 
+    override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        let contentSection = Sections.section(for: section)
+        switch contentSection {
+        case .name:
+            return content.titleErrorFooter
+        case .description:
+            return nil
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
```

Comment:
I think this will work if it just `return UITableViewAutomaticDimension` and doesn't use `footerHeight`.

I tested it just to make sure it wasn't a dumb suggestion, and it seems fine 🤞

Reply:
As it turns out that method isn't even necessary, TIL. Thanks 👍